### DISCUSSION TITLE
Condition in template

### DISCUSSION THREAD

**Discussion by Gianluks90** [sffc/card-creatr-studio#77] [Q&A] [answered]
Hi, I recently discovered Card Creator Studio and I am already in love with it. I just need to make card creation more functional without having separate files for card "category". I am not being able to use the Conditions (IF ELSE) in the Template, can you tell me which is the correct syntax?

I would like to arrive at the following solution: IF the category is "X" then it shows some icons (image loaded in Assets). Category is one of the Fields.

Thanks in advance! Gianluca

**Comment by sffc** [comment, marked as answer]
Hi @Gianluks90! The following should work

```
- if (category == "X") {
+imageFill(image1)
- } else {
+imageFill(image2)
- }
```

Note that arbitrary JavaScript code can be put on any line that starts with a `-`

Hope this helps!

**Reply by PathHound** [reply]
Can the IF statements be used to apply certain styles to text such as highlighting, emboldening or changing colors when certein words pop up? Specifically if a card term crops up you want to highlight on a card?

**Reply by sffc** [reply]
If you want to style the whole paragraph, the `if` statements work fine. If you want to style individual words within the paragraph, you can write a little code to pre-process the text and then feed it into `draft_richTextWrap` (#13).
